tensorbay/cli/cli.py

```diff
@@ -329,5 +329,25 @@ def branch(obj: Dict[str, str], tbrn: str, verbose: bool) -> None:
     _implement_branch(obj, tbrn, verbose)
 
 
+@cli.command()
+@click.argument("tbrn", type=str, default="")
+@click.argument("tag_name", type=str, default="")
+@click.option("-d", "--delete", "is_delete", is_flag=True, help="Delete the tag.")
```

Comment:
Better to add other arguments and options when implementing the corresponding commands.